Alignment/CommonAlignmentProducer/python/ALCARECOMuAlCalIsolatedMu_Output_cff.py

```diff
@@ -7,6 +7,7 @@
     ),
     outputCommands = cms.untracked.vstring(
         'keep *_ALCARECOMuAlCalIsolatedMu_*_*', 
+        'keep *_generalTracks_*_*',
```

Comment:
@anthonykrose @pakhotin
As discussed in the workshop, the `generalTracks` is *needed* for track propagation in Muon chambers. As reported by @abbiendi, you should add a selector on track transverse momentum by adding it to the AlCaReco `Path`.